docs/_posts/2019-07-31-531-release.md

```diff
@@ -0,0 +1,37 @@
+---
+layout: blogpost
+title:  "5.3.1 Release Announcement"
+section_id: blog
+date:   2019-07-31 12:00:00
+author: Open GEE Development Team
+---
+
+<br />
+
+Hi Open GEE Community!
+ 
+We are excited to announce the official release of Open GEE 5.3.1!  This release updates Open GEE with minor enhancements and bug fixes for GEE Fusion and Server.
+
+**Enhancements:**
+
+**- Uninstallation scripts are now copied to <code>/opt/google/install</code>.** This allow users to uninstall Fusion and Server without having to have a copy of the source code on hand.
+
+**- Memory and performance improvements for large builds.** Memory utilization and performance have been improved, especially for large builds.
+
+**- Increased logging during long Fusion operations.** Fusion can now be configured to log progress while calculating overlaps between resources.
+
+**- Overlay option is now supported in terrain project command-line interfaces.** <code>genewterrainproject</code>, <code>geaddterrainproject</code> and <code>gemodifyterrainproject</code> have a new option for overlay support.
+
+**- Cutter embeds extra metadata into portable files.** The bounding box metadata can be used to improve loading performance in some applications.
+
+**Supported Operating Systems:**
+
+**- Starting with Open GEE 5.3.2 Ubuntu 14.04 will not be supported any more.** Open GEE 5.3.1 is last version supporting Ubuntu 14.04.
```

Comment:
* will no longer be supported

Reply:
* Open GEE 5.3.1 will be the last version to support Ubuntu 14.04 due to its recent end of life.

Reply:
ok